import-service/handlers/importProductsFile.ts

```diff
@@ -0,0 +1,46 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { S3 } from 'aws-sdk';
+import 'source-map-support/register';
+
+const BUCKET = 'rss-task5';
+
+async function getSignedUrl(s3, params){
+    return new Promise((resolve,reject) => {
+        s3.getSignedUrl('putObject', params, (err, url) => {
```

Comment:
aws already has method that return promise

```
await s3.getSignedUrlPromise('putObject', { ...params, Expires: 60 });
```

Reply:
Thanks, will use it.